armi/reactor/tests/test_blocks.py

```diff
@@ -1708,6 +1708,10 @@ def test_getDuctPitch(self):
         ductOP = self.HexBlock.getDuctOP()
         self.assertAlmostEqual(70.6, ductOP)
 
+    def test_getPinCenterFlatToFlat(self):
+        f2f = self.HexBlock.getPinCenterFlatToFlat()
+        self.assertAlmostEqual(1.3336791218280355, f2f)
```

Comment:
any option to build this from the input data? Again we've just been bitten a lot when the test fixtures change and we have like N-hundred unit tests to go update it can be a pain.

Reply:
To be honest, I wasn't exactly sure what this is calculating, so I just copied the output it produced (fingers crossed that it's already implemented correctly). I'm not sure what "flat-to-flat" means in reference to pins. Is that just the pin pitch? I'll take a look at the numbers and see where this is coming from.

Reply:
I changed this to calculate from the input data stored on self.HexBlock. It looks a lot like the function it's testing, but still better for the future if the test fixture changes.